searchterms/searchterms.py

```diff
@@ -0,0 +1,36 @@
+
+def get_list_of_search_terms():
```

Comment:
This doesn't need to be a function - we can import this variable from this module straight away

Reply:
I don't know how to do that and the things I found don't look like good ideas (and they don't work), e.g;

from module import *